### ISSUE TITLE
Red herring: "Class XXX does not have an __init__ function defined"

### ISSUE THREAD

**Issue by gmagogsfm** [pytorch/pytorch#41278]
Unsure if this is a real issue, but this is a common error in parity bench.

```
mport torch

from torch.autograd import Variable


class MyModule(torch.nn.Module):
    def __init__(self):
        super(MyModule, self).__init__()

    def forward(self, x):
        x = Variable(x)
        return x


torch.jit.script(MyModule())
```

```
mport torch

from torch.autograd import Variable


class MyModule(torch.nn.Module):
    def __init__(self):
        super(MyModule, self).__init__()

    def forward(self, x):
        x = Variable(x)
        return x


torch.jit.script(MyModule())
```

One concrete example is [here](https://github.com/jansel/pytorch-jit-paritybench/blob/6c23f5bcbed5ad85ac21573024748b335b8019eb/generated/test_HongyangGao_Graph_U_Nets.py#L310)

I am not sure if this is a real issue as Variable should be created during Module creation time instead of in forward, thus it may just be a red herring.

cc @suo @gmagogsfm

**Comment by eellison** [comment]
This is a deprecated way of creating a tensor with gradient, Variable no longer exists. the updated idiom is `x.requires_grad(True)`

**Comment by gmagogsfm** [comment]
Closing since Variable is deprecated.
